### ISSUE TITLE
ValueError: need at most 63 handles, got a sequence of length 66

### ISSUE THREAD

**Issue by Daegalus** [AcidWeb/CurseBreaker#162]
OS: Windows 10 20.04
Terminal: CMD and Powershell 6
WoWs: Retail and Classic

So I did an Import of all my addons into Cursebreaker and all seems fine, but when I do `status` or start automatic update. I get the following error output:

```
0/51 | 
Exception in thread Thread-1:
Traceback (most recent call last):
  File "threading.py", line 932, in _bootstrap_inner
  File "threading.py", line 870, in run
  File "multiprocessing\pool.py", line 519, in _handle_workers
  File "multiprocessing\pool.py", line 499, in _wait_for_updates
  File "multiprocessing\connection.py", line 879, in wait
  File "multiprocessing\connection.py", line 811, in _exhaustive_wait
ValueError: need at most 63 handles, got a sequence of length 66
26/51 | ────────
```

It then gets stuck there indefinitely until I `ctrl-c` to exit. Same thing for Classic but with 4/9 is where it gets stuck.

**Comment by AcidWeb** [comment]
Upload yours `WTF/CurseBreaker.json`

**Comment by Daegalus** [comment]
https://ocean.yuli.dev/f/ca49e2ac96f74fc8b69b/ JSON file

And yes I do. a 2990WX with 32 cores / 64 threads

**Comment by AcidWeb** [comment]
Belive me or not but that is cause. I not declared worker limit for multithreading and apparently 60 is a limit on Windows.

I will fix it in next release.

**Comment by Daegalus** [comment]
Oh I believe you. As a dev myself, I am familiar and strange bugs like this, especially in Windows . Once you mentioned my core counts, I googled the error (which i should have done before bug reporting so I could have given a more accurate bug report). I found https://bugs.python.org/issue26903 and the related https://bugs.python.org/issue40263 which seems to make sense on why it broke. No rush, but thank you for working on fixing an edge case.
